packages/wethegit-react-hooks/src/styles.css

```diff
@@ -7,8 +7,26 @@
   opacity: 1;
 }
 
+.inViewDemo {
+  background: #ddd;
+}
+
 .isInView {
-  border: solid 1px red;
+  outline: 1px solid red;
```

Comment:
In case anyone is wondering why I made this change, it's because `outline` doesn't alter the box-model, while `border` does. This was causing issues when using a custom observer `rootMargin`